src/main/java/net/imagej/patcher/HeadlessGenericDialog.java

```diff
@@ -124,12 +124,16 @@ public void addChoice(String label, String[] items, String defaultItem) {
 		choices.add(items[index]);
 	}
 
-	public void addNumericField(String label, double defaultValue, int digits) {
+	public void addNumericField(String label, double defaultValue) {
 		numbers.add(getMacroParameter(label, defaultValue));
 	}
 
+	public void addNumericField(String label, double defaultValue, int digits) {
+		addNumericField(label, defaultValue);
+	}
+
 	public void addNumericField(String label, double defaultValue, int digits, int columns, String units) {
-		addNumericField(label, defaultValue, digits);
+		addNumericField(label, defaultValue);
```

Comment:
Shouldn't the `digits` parameter value be passed down the line here?

Reply:
`digits`, as well as `columns` and `units` in the other signature, only make sense in a non-headless environment, as they define details of how the dialog is displayed.
All that `addNumericField` does is calling:

    numbers.add(getMacroParameter(label, defaultValue));

i.e. only `label` and `defaultValue` are actually used.

I essentially kept this pattern from the two pre-existing signatures (`addNumericField(String, double, int, int, String)` was calling `addNumericField(String, double, int)` before, dropping the last two arguments).